.gitignore

```diff
@@ -21,4 +21,5 @@ src/main/antlr4/.antlr
 result.txt
 examplesold/
 src/test/scala/dump/
-src/test/analysis/dump/
\ No newline at end of file
+src/test/analysis/dump/
+-analyse
```

Comment:
What's this for?

Reply:
Resolved. [Upcoming merge from yousif-memory-region-analysis]
The `-analyse` no longer applies but the `src/test/analysis/dump/` was used when running MRA tests to generate dot output files and compare them with the `dotExpected` folder files